rts/lra-examples/cdi-embedded/src/main/java/io/narayana/rts/lra/CdiBasedResource.java.new

```diff
@@ -0,0 +1,100 @@
+package io.narayana.rts.lra;
+
```

Comment:
Looks similar to `CdiBasedResource.java`

Reply:
It's the same participant. But what is your point?

Reply:
Here I am seeing two files i.e., `CdiBasedResource.java.new` and there is another file `CdiBasedResource.java`.